Regression/WarpX-tests.ini

```diff
@@ -1657,24 +1657,6 @@ compareParticles = 1
 particleTypes = electrons beam
 analysisRoutine = Examples/analysis_default_regression.py
 
-[Python_LaserAccelerationMR]
```

Comment:
This test was associated with the old PICMI script that was removed, and I think it was not using mesh refinement (despite what the name tag `MR` may suggest).